objectstorage.go

```diff
@@ -66,9 +67,12 @@ func (c *Client) GetObjectStorageAccessKeyList() ([]ObjectStorageAccessKey, erro
 }
 
 //GetObjectStorageAccessKey gets a specific object storage access key based on given id
-func (c *Client) GetObjectStorageAccessKey(id string) (ObjectStorageAccessKey, error) {
+func (c *Client) GetObjectStorageAccessKey(accessKey string) (ObjectStorageAccessKey, error) {
```

Comment:
In API docs, the name of the object's ID might change based on the resource name e.g,(iso_image_uuid, server_uuid, access_keys). I would keep `id` to keep consistency in gs-client.

Reply:
But you can only use access key to access an object storage.

Reply:
`accessKey` is ID of firewall. All `Get` a resouce should use `id`.

Reply:
> `accessKey` is ID of firewall. All `Get` a resouce should use `id`.

I couldn't find any information on our API docs about "`accessKey` is ID of firewall".